### PR TITLE
upgrade to boto3

### PR DESCRIPTION
**Pull request by chegejames**

### CODE REVIEW

**Comment by DavidLemayian** [comment]
@chegejames Missing it in the `requirements.txt`?

**Comment by chegejames** [comment]
@DavidLemayian I've updated the requirements.txt

File: requirements.txt [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+jmespath==0.9.3
 Logbook==1.0.0
 MySQL-python==1.2.5
 oauthlib==2.0.1
+python-dateutil==2.6.0
 pytz==2016.10
 PyYAML==3.12
 requests==2.12.4
 requests-oauthlib==0.7.0
+s3transfer==0.1.10
```
Comment (DavidLemayian) [inline requirements.txt:16]: @chegejames Are all of these necessary or just boto3?

Comment (chegejames) [reply requirements.txt:16]: @DavidLemayian The rest are collected packages from boto3. I did a pip freeze.

**Pull request merged**